client/src/app/app.component.ts

```diff
@@ -1,64 +1,71 @@
-                rationale: "A rationale",
-                url: "https://www.notion.so/Child-2-abcdefghi",
-            },
-            {
-                id: "qwertyuiop",
-                parent: "123456789",
-                title: "UTAT-Team-D",
-                qualifier: "SHOULD",
-                type: "A type",
-                description: "Yet another description that is just a little bit longer",
-                rationale: "A rationale",
-                url: "https://www.notion.so/Grandchild-1-qwertyuiop",
-            },
-        ],
-    })
+  public model: go.TreeModel = new go.TreeModel({
+    nodeKeyProperty: "id",
+    nodeDataArray: [
+        {
+            id: "ff5892a7-b5c9-4a25-97bd-663b2cff5bd7",
+            parent: "",
+            title: "UTAT-Team-A",
+            qualifier: "SHALL",
+            type: "Performance",
+            description: "A description",
+            rationale: "A rationale",
+            url: "https://www.notion.so/UTAT-Team-A-74331bc1129c417c850a0a5ca0deb759",
+        },
+        {
+            id: "123456789",
+            parent: "ff5892a7-b5c9-4a25-97bd-663b2cff5bd7",
+            title: "UTAT-Team-B",
+            qualifier: "SHOULD",
+            type: "Constraint",
+            description: "Another description",
+            rationale: "A rationale",
+            url: "https://www.notion.so/Child-1-123456789",
+        },
+        {
+            id: "abcdefghi",
+            parent: "ff5892a7-b5c9-4a25-97bd-663b2cff5bd7",
+            title: "UTAT-Team-C",
+            qualifier: "SHALL",
+            type: "Component Selection",
+            description:
+                "A very long form description that may or may not exceed the width of the node. The quick brown fox jumps over the lazy dog. Octopuses have several tentacles. Icecream is cold. Did you know that lava is warm? Lava cake with an ice cream scoop on the side is delicious.",
+            rationale: "A rationale",
+            url: "https://www.notion.so/Child-2-abcdefghi",
+        },
+        {
+            id: "qwertyuiop",
+            parent: "123456789",
+            title: "UTAT-Team-D",
+            qualifier: "SHOULD",
+            type: "A type",
+            description: "Yet another description that is just a little bit longer",
+            rationale: "A rationale",
+            url: "https://www.notion.so/Grandchild-1-qwertyuiop",
+        },
+    ],
+  });
 
-    public setSelectedNode(node: go.Part | null) {
-        this.selectedNode = node
-    }
-}
+  public setSelectedNode(node: go.Node) {
+    this.selectedNode = node;
+    console.log(this.selectedNode)
+  }
+
+  public setSelectedNode2(node: go.Node) {
+    this.selectedNode2 = node;
+    console.log(this.selectedNode2)
+  }
```

Comment:
Could you provide some insight as to the use case for each of these selected node properties? Is there a reason we couldn't have a single selected node?